Mising no_log for create user and password

ricolin opened the pull request:
Change-Id: I9b60ca505d7fd6ab6462fcc0825bbfbee4de2d9f

mnaser reviewed (changes requested):
@ricolin I think this should include a release note.

ricolin commented:
> @ricolin I think this should include a release note.

updated

Commit:
fix: hide rook ceph user credentials from logs

Add no_log to the Rook Ceph OpenStack user creation task so the
username/password module arguments are not emitted in Ansible logs.

Change-Id: I45b714cdb1df8e2ac852e6b8852cbf901c6447cf
Signed-off-by: Rico Lin <rico@vexxhost.com>
Assisted-By: OpenAI Codex <noreply@openai.com>
